### PR TITLE
formatted docstring of sympy.geometry.curve according to documentation

### PR DESCRIPTION
**Pull request by viratkohli1969**
#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/geometry/curve.py [hunk 1]
```diff
@@ -91,33 +92,28 @@ def _eval_subs(self, old, new):
...
-        """
-        A parameterized point on the curve.
+        """A parameterized point on the curve.
 
         Parameters
         ==========
 
         parameter : str or Symbol, optional
-            Default value is 't';
+            Default value is 't'.
```
Comment (smichr) [inline sympy/geometry/curve.py:101]: The 'the' in the next line should be capitalized now.

File: sympy/geometry/curve.py [hunk 2]
```diff
@@ -148,10 +149,15 @@ def arbitrary_point(self, parameter='t'):
...
-        """
-        Return a set of symbols other than the bound symbols used to
+        """Return a set of symbols other than the bound symbols used to
         parametrically define the Curve.
 
+        Returns
+        =======
+
+        set :
+            Set of all non parameterized symbols.
```
Comment (smichr) [inline sympy/geometry/curve.py:159]:
```suggestion
            Set of all non-parameterized symbols.
```

File: sympy/geometry/curve.py [hunk 3]
```diff
@@ -170,6 +177,25 @@ def free_symbols(self):
 
     @property
     def ambient_dimension(self):
+        """The dimension of the curve.
+
+        Returns
+        =======
+
+        int:
+            returns the dimension of curve.
```
Comment (smichr) [inline sympy/geometry/curve.py:186]:
```suggestion
            the dimension of curve.
```

File: sympy/geometry/curve.py [hunk 4]
```diff
@@ -170,6 +177,25 @@ def free_symbols(self):
 
     @property
     def ambient_dimension(self):
+        """The dimension of the curve.
+
+        Returns
+        =======
+
+        int:
```
Comment (smichr) [inline sympy/geometry/curve.py:185]:
```suggestion
        int :
```

File: sympy/geometry/curve.py [hunk 5]
```diff
@@ -232,12 +259,8 @@ def parameter(self):
         Returns
         =======
 
-        parameter : SymPy symbol
-
-        See Also
-        ========
-
-        functions
+        Symbol:
```
Comment (smichr) [inline sympy/geometry/curve.py:262]:
```suggestion
        Symbol :
```
(not sure -- some have space and some don't)

File: sympy/geometry/curve.py [hunk 6]
```diff
@@ -280,13 +309,9 @@ def plot_interval(self, parameter='t'):
...
 
-        plot_interval : list (plot interval)
-            [parameter, lower_bound, upper_bound]
-
-        See Also
-        ========
-
-        limits : Returns limits of the parameter interval
+        List :
+            returns a list(plot interval) as below:
```
Comment (smichr) [inline sympy/geometry/curve.py:313]:
```suggestion
            the plot interval as below:
```

File: sympy/geometry/curve.py (hunk 5 above)
Comment (mohitshah3111999) [reply sympy/geometry/curve.py:262]: I think providing space to the docstrings will make it more readable.

**Comment by smichr** [comment]
It's in. Thanks.

**Pull request merged**